department-of-veterans-affairs/abd-vro
# VRO Custom Redis and RabbitMQ

## What was the problem?

Use of third-party images were not passing SecRel and blocking deployment of the VRO platform helm chart.

Associated tickets or Slack threads:
- #1007 

## How does this fix it?[^1]
Publishes custom versions of the bitnami images and replaces the use of the mirrored third-party images with the custom images. 

## How to test this PR
- https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/6647464293 passing secrel run
- successful deployment to dev environment https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/6647699170


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
